QgisModelBaker/libqgsprojectgen/generator/generator.py

```diff
@@ -462,7 +462,7 @@ def relations(self, layers, filter_layer_list=[]):
 
     def full_node(self, layers, item):
         current_node = None
-        if item:
+        if isinstance(item, dict):
```

Comment:
I think an empty dictionary will return `True`, is this expected?

Reply:
You are right. It should never be empty but still better to check it.